lib/services/resource.go

```diff
@@ -613,3 +787,43 @@ func (r *Ref) Set(v string) error {
 func (r *Ref) String() string {
 	return fmt.Sprintf("%s/%s", r.Kind, r.Name)
 }
+
+// Refs is a set of resource references
+type Refs []Ref
+
+// ParseRefs parses a comma-separated string of resource references (eg "users/alice,users/bob")
+func ParseRefs(refs string) (Refs, error) {
+	isComma := func(r rune) bool { return r == ',' }
+	var parsed []Ref
+	split := strings.FieldsFunc(strings.TrimSpace(refs), isComma)
```

Comment:
Do we currently make the explicit assertion elsewhere in Teleport that a comma can never be part of a resource name? Just trying to make sure we don't ever run into an issue where someone called their role "admins,1" which might break this.

Reply:
Hmm, good catch.  I don't think we currently have any means to enforce a requirement that broad; different resources have different rules.  I suppose supporting backslash-escaping should cover the usability issue (e.g. `tctl get 'role/admins\,1'`), though in the future I it would be nice to globally enforce an allowed charset for resource names.

Reply:
So it turns out we do have an enforcement point (kinda).  The backend restricts the charset of all keys, which indirectly restricts the charset of resource names.  Looks like commas are rejected.  Its not foolproof though; you can still create a resource with `/` in its name, which `tctl` will not be able to parse correctly.